### DISCUSSION TITLE
DataArray to Zarr

### DISCUSSION THREAD

**Discussion by skgbanga** [pydata/xarray#5128] [Q&A] [answered]
Currently, there is a way in which a `Dataset` can be stored in zarr and it is described at:
http://xarray.pydata.org/en/stable/io.html?highlight=zarr#zarr

Is there a way to store `DataArray` into zarr directly as well? 

PS: One can always create a Dataset with a single element:
> xr.Dataset({"bar": foo})

where foo is a DataArray to accomplish the above. 

I was wondering if this is the recommended way.

**Comment by max-sixty** [comment]
Hi @skgbanga  — yes that's currently the recommended way.

Edit: I found a way to convert this to a discussion, which is a much better match for this sort of question.

**Comment by keewis** [comment, marked as answer]
you could also use the [`to_dataset`](https://xarray.pydata.org/en/latest/generated/xarray.DataArray.to_dataset.html) method:
```python
foo.to_dataset()
# or, if foo.name is None
foo.to_dataset(name="bar")
```
